src/pages/BusinessSpecifics/BusinessSpecifics.tsx

```diff
@@ -2,14 +2,35 @@ import { FormControl, TextField, Button, MenuItem, Select } from "@mui/material"
 import { useState } from "react"
 import { currencies } from "./currencies"

```

Comment:
we need to move this file into the constants directory

Reply:
should all of the forms be in constants?

Reply:
only the currencies file